MapboxMobileEventsTests/MMEEventsManagerTests.mm

```diff
@@ -829,20 +829,6 @@ @interface MMEEventsManager (Tests) <MMELocationManagerDelegate>
                 });
             });
             
-            context(@"when a carplay event is pushed", ^{
```

Comment:
Why delete this test? It seems it should be updated for the new event name?

Reply:
It's no longer needed since the prefix check for `navigation` already exists